Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -50,8 +50,8 @@
             <Inventory objectType="Steel Plate" amount="1" />
         </DeconstructJob>
 
-        <OnUpdate FunctionName="OnUpdate_Door" />
-        <OnUpdate FunctionName="OnUpdate_Leak_Door" />
+        <Action event="OnUpdate" function_name="OnUpdate_Door" />
```

Comment:
I think that we should use camelCase for the attribute names since it used for all the other attributes. ("functionName" instead of "function_name")